app/controllers/api/v0/followers_controller.rb

```diff
@@ -5,15 +5,15 @@ class FollowersController < ApiController
       before_action -> { limit_per_page(default: 80, max: 1000) }
 
       USERS_ATTRIBUTES_FOR_SERIALIZATION = %i[
-        id follower_id follower_type
+        id follower_id follower_type created_at
       ].freeze
       private_constant :USERS_ATTRIBUTES_FOR_SERIALIZATION
 
       def users
         @follows = Follow.followable_user(@user.id)
           .includes(:follower)
           .select(USERS_ATTRIBUTES_FOR_SERIALIZATION)
-          .order(created_at: :desc)
+          .order(created_at: sort_param)
```

Comment:
It looks like `created_at` is the sort param here, and this is actually the sort direction.

```suggestion
          .order(created_at: sort_direction)
```

Reply:
Well, it is the param called `sort` but that doesn't change the fact that it was a horrible method name. 😆 Thanks!